packages/site/pages/components/avatar.js

```diff
@@ -55,28 +57,82 @@ export default _ => (
         ]}
       />
 
-      <SectionHeading>Size</SectionHeading>
-      <P>Five avatar sizes are available.</P>
-      <Example.React
-        includes={{ Avatar }}
-        codes={Object.keys(Avatar.sizes).map(
-          size =>
-            `<Avatar size={Avatar.sizes.${size}} src="https://secure.gravatar.com/avatar/0f792a763ebf08411c7f566079e4adc7?s=400" />`
-        )}
-      />
-
-      <SectionHeading>Default with initials</SectionHeading>
+      <SectionHeading>Sizes and fallback states</SectionHeading>
+      <P>
+        Avatars are available in five sizes specified with the <code>size</code>{' '}
+        prop.
+      </P>
       <P>
-        An avatar with a visual image is preferred. In cases where the image is
-        unavailable or loading, initials of the person pictured will be
-        displayed. Always provide the <code>name</code> prop as a fallback.
+        An avatar with an image is preferred. In cases where the image is
+        unavailable or loading, the first initial of the person pictured will be
+        displayed on a background of nine possible colors. Always provide the{' '}
+        <code>name</code> prop as a fallback.
       </P>
       <Example.React
         includes={{ Avatar }}
+        outputStyle={{
+          display: 'grid',
+          gap: core.layout.spacingXXLarge,
+          gridTemplateColumns: `auto auto auto auto auto`,
+          alignItems: 'center'
+        }}
+        outputChildStyle={{ margin: 0 }}
         codes={[
-          `<Avatar name="Alan Turing" />`,
-          `<Avatar name="Grace Hopper" />`,
-          `<Avatar name="Tim Berners-Lee" />`
+          `<Avatar size="xLarge" name="Assis Silva" src="https://secure.gravatar.com/avatar/0f792a763ebf08411c7f566079e4adc7?s=400"  />`,
```

Comment:
opinion(non-blocker): Showing the permutations seems like a fine idea to me.  Just personally, it feels like too much.  Also, I have to scroll 2 pages before I can see what code generates the visuals, and it's hard to pair code to visuals.

Reply:
Good feedback. I adjusted the examples in the docs.